i18n-helpers/Cargo.toml

```diff
@@ -19,7 +19,7 @@ mdbook.workspace = true
 polib.workspace = true
 pulldown-cmark = { version = "0.9.2", default-features = false }
 pulldown-cmark-to-cmark = "11.2.0"
-regex = "1.10.4"
+regex = "1.9"
```

Comment:
We might get a PR from the dependabot, but we can then tell it to ignore the 1.10 releases.